src/frontend/apps/.cookiecutter/{{cookiecutter.app_name}}/data/{{cookiecutter.app_name}}AppData.ts

```diff
@@ -0,0 +1,5 @@
+import { parseDataElements } from 'apps/{{cookiecutter.app_name}}/utils/parseDataElements/parseDataElements';
+
+export const {{cookiecutter.app_name}}AppData = parseDataElements(
+  document.getElementById('marsha-frontend-data')!,
```

Comment:
Every app and the lti site uses the same dom content ? (And yet, we will remove this assert one day)

Reply:
Yes all apps user the same node.